src/System.Runtime.Numerics/tests/ComplexTests.cs

```diff
@@ -410,13 +440,40 @@ public static IEnumerable<object[]> ASin_Advanced_TestData()
 
         [Theory, MemberData("ASin_Advanced_TestData")]
         [SkipOnTargetFramework(TargetFrameworkMonikers.NetFramework)]
-        public static void ASin_Advanced(double real, double imaginary, double expectedReal, double expectedImaginary)
+        public static void ASin_Advanced (double real, double imaginary, double expectedReal, double expectedImaginary) {
```

Comment:
There also shouldn't be a space between the method name and the opening parenthesis (we don't follow that convention elsewhere in the document).